options/linux/generic/pty-stubs.cpp

```diff
@@ -15,10 +15,6 @@
 int openpty(int *mfd, int *sfd, char *name, const struct termios *ios, const struct winsize *win) {
 	__ensure(!ios);
```

Comment:
Might be nice to also implement this. It should probably be placed alongside the TIOCSWINSZ.
```cpp
if(ios)
    tcsetattr(ptmx_fd, TCSAFLUSH, ios);
```